Task2/js/app.js

```diff
@@ -0,0 +1,24 @@
+document.addEventListener('DOMContentLoaded', () => {
+    const container = document.querySelector('.zombies');
+
+    for (let i = 0; i < zombies.length; i++) {
+        const zombie = document.createElement('div');
+
+        zombie.classList.add('zombie');
+
+        zombie.addEventListener('click', () => {
+            zombie.classList.remove(zombie.classList[zombie.classList.length - 1]);
+            zombie.classList.add('dead');
+        });
+
+        if (zombies[i]['type'] === ZOMBIE_TYPE.SMALL) {
```

Comment:
zombies[i]['type']
really?

not just 
zombies[i].type
?

what's the point of using ['']
?

Reply:
as for me it’s more convenient to use either only square brackets or only dots, where it is possible))